migration-roadmap/status-light.md

```diff
@@ -0,0 +1,259 @@
+| `.spectrum-StatusLight--cinnamon`   | `variant="cinnamon"`   | Missing from WC - new for S2                                                              |
+| `.spectrum-StatusLight--brown`      | `variant="brown"`      | Missing from WC - new for S2                                                              |
+| `.spectrum-StatusLight--silver`     | `variant="silver"`     | Missing from WC - new for S2                                                              |
+| `.spectrum-StatusLight--gray`       | `variant="gray"`       | Missing from WC, not in the design spec for S2                                            |
+| `.spectrum-StatusLight--blue`       | `variant="blue"`       | Missing from WC, not in the design spec for S2 (appears to be different from "info")      |
+| `.spectrum-StatusLight--green`      | `variant="green"`      | Missing from WC, not in the design spec for S2 (appears to be different from "positive")  |
+| `.spectrum-StatusLight--orange`     | `variant="orange"`     | Missing from WC, not in the design spec for S2 (appears to be different from "notice")    |
+| `.spectrum-StatusLight--red`        | `variant="red"`        | Missing from WC, not in the design spec for S2 (appears to be different from "negative" ) |
+| `.spectrum-StatusLight--accent`     | `variant="accent"`     | Missing from WC, appears to be deprecated for S2                                          |
+
+## Summary of changes
+
+### CSS => SWC changes
+
+The status light component has several missing variants in the web component implementation:
+
+**Missing from WC:**
+
+**New variants for S2:**
+
+- Pink variant
+- Turquoise variant
+- Cinnamon variant
+- Brown variant
+- Silver variant
+
+**Variants present in both S1 and S2:**
+
+- Cyan variant: CSS supports `--cyan` but web components doesn't, this is in the design spec and **will need to be added in 2nd gen**
+
+**Variants not in design spec for S2 (need verification):**
+These variants are not in the design spec for S2 but are present in the S2 preview for CSS. These will likely need verification to determine whether or not we will include them.
+
+Notes: These variants all exist in the S2 design spec and CSS preview of the Badge component. These all also have a similarly-colored "semantic" variant. This may be useful when determining whether they should be included in the 2nd gen web components.
+
+- Gray variant (similar to "neutral" variant)
+- Blue variant (similar to "info" variant)
+- Green variant (similar to "positive" variant)
+- Orange variant (similar to "notice" variant)
+- Red variant (similar to "negative" variant)
+
+**Deprecated variants:**
+
+- Accent variant: deprecated for S2
+
+**Other implementation differences:**
+
+- **Disabled attribute**: There does not appear to be a disabled state for this component in the S2 design spec, we can likely deprecate this attribute for 2nd gen web components.
```

Comment:
I checked on this last year: https://github.com/adobe/spectrum-css/pull/3153#issuecomment-2447559551

Disabled will not be in S2 at this time, and should be deprecated.